lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -5,8 +5,39 @@
 // @dart = 2.10
 part of engine;
 
-/// EXPERIMENTAL: Enable the Skia-based rendering backend.
-const bool experimentalUseSkia =
+/// A JavaScript entrypoint that allows developer to set rendering backend
+/// at runtime before launching the application.
+@JS('window.flutterWebRenderer')
+external String? get requestedRendererType;
+
+/// Whether to use CanvasKit as the rendering backend.
+bool get useCanvasKit =>
+    _autoDetect ? _detectRenderer() : _useSkia;
+
+/// Returns true if CanvasKit is used.
+/// Otherwise, returns false.
```

Comment:
very nit: I believe there is one empty line between two sentences in doc comments.  https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph